lib/usd/hdMaya/adapters/materialAdapter.cpp

```diff
@@ -317,7 +325,29 @@ class HdMayaShadingEngineAdapter : public HdMayaMaterialAdapter
         }
     }
 
-#if USD_VERSION_NUM <= 1911
+#if USD_VERSION_NUM <= 2011
+
+    HdTextureResourceSharedPtr GetTextureResource(const SdfPath& textureShaderId) override
+    {
+        TF_DEBUG(HDMAYA_ADAPTER_MATERIALS)
+            .Msg(
+                "HdMayaShadingEngineAdapter::GetTextureResource(%s): %s\n",
+                textureShaderId.GetText(),
+                GetID().GetText());
+        if (GetDelegate()->IsHdSt()) {
+            auto* mObjPtr = TfMapLookupPtr(_materialPathToMobj, textureShaderId);
+            if (!mObjPtr || (*mObjPtr) == MObject::kNullObj) {
+                return {};
+            }
+            return GetFileTextureResource(
+                *mObjPtr,
+                GetFileTexturePath(MFnDependencyNode(*mObjPtr)),
+                GetDelegate()->GetParams().textureMemoryPerTexture);
+        }
+        return {};
+    }
+
+#elif USD_VERSION_NUM <= 1911
```

Comment:
this elif branch will never be entered - see above note

Reply:
Addressed by d9ea7b2abc0f27f79638e3602c129a9812e73fd1.